xrpl4j-core/src/test/java/org/xrpl/xrpl4j/model/transactions/IssuedCurrencyAmountTest.java

```diff
@@ -0,0 +1,51 @@
+package org.xrpl.xrpl4j.model.transactions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit tests for {@link IssuedCurrencyAmount}.
+ */
+class IssuedCurrencyAmountTest {
+
+  @Test
+  void toStringIssuedCurrentAmount() {
+    // Negative values.
+    IssuedCurrencyAmount issuedCurrency = IssuedCurrencyAmount.builder()
+      .currency("USD")
+      .issuer(Address.of("rP9JR5JTEqaVYbXHtiqR5YvBeoWQeMBipS"))
+      .value("-1.00")
+      .build();
+    assertThat(issuedCurrency.toString()).isEqualTo(
+      "IssuedCurrencyAmount{" +
+        "value=-1.00, " +
```

Comment:
See my comment above about whether this should emit as a negative string, or a positive one. In rippled JSON RPC responses, these emit as negative values (there is no `isNegative` in there) so from that perspective, this format is more aligned. However, it does feel weird to have a negative number with an `isNegative` property.

Maybe we exclude `isNegative` from the JSON of `IssuedCurrencyAmount` and that will make things clearer.

Reply:
I wonder if `isNegative` should live in `CurrencyAmount` or if it should just be a property of `XrpCurrencyAmount`...

If we want to keep it in `CurrencyAmount`, I think it's fine to include it in the `toString` implementation.

Reply:
Agreed. I lean slightly towards leaving `isNegative` in the top-level interface, so resolving this one for now. I created #547 for later tracking.